assets/css/dup_v2.scss

```diff
@@ -12,6 +12,7 @@
 
 @import "v2/placeholder";
 @import "v2/dup/normal_header";
+@import "v2/dup/evergreen/image";
```

Comment:
Only `image.scss` is imported here, not `video.scss`. 

Since the two scss files are essentially identical, can they be consolidated into just `evergreen.scss`?

Reply:
Good idea. I don't see `image` and `video` ever needing different dimensions so separating them is unnecessary.